static/js/Router.js

```diff
@@ -1,29 +0,0 @@
-import React from 'react';
```

Comment:
Just curious, is this file and the `react-router` module's functionality replaced by material-components?